### PR TITLE
Add release highlight for #106133

### PR DESCRIPTION
**Pull request by ChrisHegarty**
This commit adds a release highlight for #106133.

### CODE REVIEW

**Comment by ChrisHegarty** [comment]
I dunno if we want to, or how appropriate it is, but we could link to the blog https://www.elastic.co/search-labs/blog/vector-similarity-computations-ludicrous-speed

File: docs/changelog/106133.yaml [hunk 1]
```diff
@@ -6,11 +6,13 @@ issues: []
 highlight:
   title: A SIMD (Neon) optimised vector distance function for Scalar Quantized int8 has been added
```
Comment (benwtrent) [inline docs/changelog/106133.yaml:7]: we should be very specific that this is for merges only.

**Review by benwtrent — APPROVED** [review]
Reads ok to me. Just want to be very specific that this is for merges only

**Pull request merged**